tikv-client/src/test/java/com/pingcap/tikv/predicates/ScanAnalyzerTest.java

```diff
@@ -25,7 +25,6 @@
 import com.google.common.collect.ImmutableSet;
 import com.google.common.collect.Range;
 import com.google.protobuf.ByteString;
-import com.pingcap.tikv.codec.KeyUtils;
```

Comment:
can we check unsued import in checkstyle?

Reply:
I think it was introduced by another pr which did not formatted correctly.

Reply:
https://internal.pingcap.net/jira/browse/TISPARK-100